index.js

```diff
@@ -5,12 +5,14 @@ var React = require('react');
 var EXT_REGEX = new RegExp('\\.jsx$');
 var DEFAULTS = {
     doctype: '<!DOCTYPE html>',
-    renderMethod: 'renderToStaticMarkup',
-    layoutMethod: 'renderToStaticMarkup',
+    viewRenderMethod: 'renderToStaticMarkup',
+    layoutRenderMethod: 'renderToStaticMarkup',
+    layoutKeyword: 'content',
```

Comment:
If I understand the configuration here correctly, we are setting a default to `compileOptions.layoutKeyword`. According to the documentation hapi `server.views` options object typically defines `layoutKeyword`: http://hapijs.com/api#serverviewsoptions . This might be a bit confusing to the user of this library.

As far as I can tell, there is no way of accessing the hapi views options from the `{view engine}.compile` method without coupling the view engine to the Hapi.server instance.

I would either rename the layoutKeyword property to something else so it doesn't get confused with the hapi property of the same name, or document this very clearly.